GliaWidgets/Interactor/Interactor.swift

```diff
@@ -264,8 +265,16 @@ extension Interactor: CoreSdkClient.Interactable {
     }
 
     var onEngagementRequest: CoreSdkClient.RequestOfferBlock {
+        print("Engagement request received")
         return { [weak self] answer in
-            answer(self?.visitorContext, true) { _, _ in }
+            let completion: CoreSdkClient.SuccessBlock = { _, error in
+                if let reason = error?.reason {
+                    print(reason)
```

Comment:
let's use `debugPrint` here also, or better to keep it as `print` for now for better visibility to integrator?

Reply:
[682819e](https://github.com/salemove/ios-sdk-widgets/pull/442/commits/682819e8bbf2dda3a7c51b7e9495e5c8761701d2)

Reply:
For consistency, let's use `debugPrint` for now